build/upload.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# upload merged
+if [ -z "${AWS_ACCESS_KEY_ID}" ]; then
```

Comment:
Is this an environment variable?

Reply:
yes, it's only available when you merge though to keep it secret